packages/gatsby-theme-carbon/src/components/Homepage/Banner.module.scss

```diff
@@ -0,0 +1,25 @@
+.column {
+  @include carbon--type-style('quotation-01');
+  color: $inverse-01;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 100%;
+}
+
+.row {
+  height: 100%;
+  h1 {
```

Comment:
Super nitpicky, but can we un-nest these rules, to follow what we've been trying to do across Carbon codebase. Makes readability easier, and easier for future devs when looking and editing.